patcher/ServerListEditor.cs

```diff
@@ -43,8 +43,13 @@ private void saveButton_Click(object sender, EventArgs e)
                 list.Add(new GameServer { ServerName = item.Cells[0].Value.ToString(), ServerAddress = item.Cells[1].Value.ToString() });
             }
             GameServers = list.ToArray();
+            string appData = Environment.GetFolderPath(Environment
+                .SpecialFolder
+                .ApplicationData);
+
+            var folder = $@"{appData}\PeacockProject\";
```

Comment:
Same here.

Reply:
I also feel there should just be a single static readonly in a file with the path to the configuration and `server.json`.